reactive/kubernetes_master.py

```diff
@@ -906,6 +885,9 @@ def send_data():
         'kubernetes.default.svc.{0}'.format(domain)
     ]
 
+    # NB: this section can't be refactored to use
```

Comment:
This makes me sad. Maybe we should refactor the function to return a list and have the places that don't want more than one do the selection based on unit? Or maybe an optional param to say "return everything"?
`def get_api_endpoints():
<contents of get_api_endpoint minus culling>

def get_api_endpoint():
  ips = get_api_endpoints()
  return ips[kubernetes_common.get_unit_number() % len(ips)]